src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
   margin: 0;
+
+  --text-color-second: #00acdc;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: 100%;
+  text-transform: uppercase;
```

Comment:
Text transformation is correctly handled using CSS with `text-transform: uppercase;`. This is in line with the task requirements.